CHANGELOG.adoc

```diff
@@ -24,11 +24,12 @@ A release with an intentional breaking changes is marked with:
 ** {issue}559[#559]: Make `get-active-element` a public API. This was previously private. ({person}dgr[@dgr])
 ** {issue}629[#629]: Added `fill-human-active` as a convenience function that fills the currently active input using `fill-human-el`. ({person}dgr[@dgr])
 ** {issue}620[#620]: Get stricter when unwrapping elements. ({person}dgr[@dgr])
+** {issue}647[#647]: Fix logic bug in `intersects?`. Added tests to test suite. ({person}dgr[@dgr])
+** {issue}649[#649]: When supplied a vector argument for `q-text`, `fill-multi` and `fill-human-multi` now fill fields in the order that fields appear in the vector. Previously, the order was not guaranteed. ({person}dgr[@dgr])
 * Docs
 ** {issue}559[#559]: Deprecate use of `:active` with `query` and other APIs that use `query` under the hood. ({person}dgr[@dgr])
 ** {issue}642[#642]: Add `driver-type` to retrieve driver type keyword. ({person}dgr[@dgr])
 ** {issue}644[#644]: Deprecate `when-predicate` and `when-not-predicate` macros. See docstrings for guidance on alternatives. These will may be removed from the API at the next release that contains breaking changes. ({person}dgr[@dgr])
```

Comment:
Dave, are any of these strictly doc changes?

Reply:
Adding `driver-type` is not just a doc change. The others largely are (in the sense that adding `^:deprecated` is a doc change.

Reply:
Let me go through and check each one carefully and submit a change set to the PR.

Reply:
OK, try that. I moved to the correct section and put them in numerical order.

Reply:
Sorry @dgr, I'm going to make you suffer one more time! 
I think `deprecating` an fn is more than a doc change and, therefore, shouldn't be under doc changes. (If there are no doc changes left, feel free to delete the doc item in the list.)

Reply:
Removed.